### ISSUE TITLE
Gatsby 5 Peer Dependency Support?

### ISSUE THREAD

**Issue by broksonic21** [anantoghosh/gatsby-plugin-purgecss#1018]
**Verify if it is a postcss bug**
First verify that gatsby-plugin-purgecss is working by setting the option `printRejected: true` which will print out the list of removed selectors.
If the plugin is working, but the css is not getting purged as you would have hoped then it's very likely a purgecss issue. File for purgecss related bugs here https://github.com/FullHuman/purgecss/issues.

- [X] **I have verified that this is not a purgecss issue.**

**Describe the bug**
Gatsby 5 is out, but gatsby-plugin-purgecss only supports `gatsby@"^2.0.0 || ^3.0.0 || ^4.0.0"`

**To Reproduce**
Sample package.json to repro:

```{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "gatsby-plugin-purgecss": "^6.1.2",
    "gatsby": "^5.1.0"
  }
}
```

Error with npm:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: test@1.0.0
npm ERR! Found: gatsby@5.1.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^5.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^2.0.0 || ^3.0.0 || ^4.0.0" from gatsby-plugin-purgecss@6.1.2
npm ERR! node_modules/gatsby-plugin-purgecss
npm ERR!   gatsby-plugin-purgecss@"^6.1.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Configs**
- [X] **I have removed all personal information.**

**Expected behavior**
Support latest gatsby version as a peer dependency

**Additional information**
n/a

**Comment by anantoghosh** [comment]
I'll update it soon.

**Comment by HKdAlex** [comment]
What be really great, thank you!

**Comment by anantoghosh** [comment]
Released in v6.2.0.
